projekt_do_pracy_na_labach/fiszki_backend/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backend-deployment
+  namespace: apka
+  labels:
+    app: backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: backend
+  template:
+    metadata:
+      labels:
+        app: backend
+    spec:
+      containers:
+      - name: laby2  
+        image: kube:v1
+        imagePullPolicy: Never
+        ports:
+        - containerPort: 8000
+
```

Comment:
Brakuje ci odwołania do persitent volume aby logo się zapisywały, jak dodasz to wstawię 4.0 :)